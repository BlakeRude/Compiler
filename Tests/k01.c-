main()
{
     for i = 300 to 313 by 2 do {
         output(i);
         outnl();
     }
}
