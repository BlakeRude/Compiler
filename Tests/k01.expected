Number of warnings: 0
Number of errors: 0
------------------------------------
Loading file: k01.tm
300
302
304
306
308
310
312
Bye.
